xplique/attributions/base.py

```diff
@@ -53,13 +54,16 @@ class BlackBoxExplainer(ABC):
     _cache_models: Dict[Tuple[int, int], tf.keras.Model] = {}
 
     def __init__(self, model: Callable, batch_size: Optional[int] = 64,
-                operator: Optional[Callable[[tf.keras.Model, tf.Tensor, tf.Tensor], float]] = None):
+                operator: Optional[Union[Tasks, str, OperatorSignature]] = None):
```

Comment:
Are tasks also needed if we already support str?

Reply:
Yes as someone could import them from commons or use the enum